lib/new_relic/agent/new_relic_service.rb

```diff
@@ -471,9 +473,14 @@ def handle_error_response(response, endpoint)
         when Net::HTTPGone
           handle_gone_response(response, endpoint)
         else
-          record_endpoint_attempts_supportability_metrics(endpoint)
-          record_error_response_supportability_metrics(response.code)
-          raise UnrecoverableServerException, "#{response.code}: #{response.message}"
+          # Net::WriteTimeout requires Ruby 2.6+
```

Comment:
I understand why it's like this with the if/else inside of the case else, but maybe adding the todo I mentioned above would be good to do here as well to make sure it's not missed when this can be updated when 2.5 is dropped (someday)

Reply:
TODOs added with [607f6d5](https://github.com/newrelic/newrelic-ruby-agent/pull/2147/commits/607f6d5c9466bffb725d9151205fd411637f9ad9)